sql-ballerina/tests/simple-params-query-test.bal

```diff
@@ -552,23 +552,24 @@ function queryTimeStringInvalidParam() {
     record{}|error? result = trap queryMockClient(simpleParamsDb, sqlQuery);
     test:assertTrue(result is error);
 
-    if (result is ApplicationError) {
-        test:assertTrue(result.message().startsWith("Error while executing SQL query: SELECT * from " +
-                "DateTimeTypes WHERE time_type =  ? . java.lang.IllegalArgumentException"));
+    if (result is DatabaseError) {
+        test:assertTrue(result.message().startsWith("Error while executing SQL query: SELECT * from DateTimeTypes " +
+        "WHERE time_type =  ? . data exception: invalid datetime format."));
     } else {
-        test:assertFail("ApplicationError Error expected.");
+        io:println(result);
+        test:assertFail("DatabaseError Error expected.");
```

Comment:
shall we append the result to the assertFail message and remove `io:println`

Reply:
removed in https://github.com/ballerina-platform/module-ballerina-sql/pull/161/commits/1a363a5654c30b1c3b4efd836eb9225713c4e5a6